datafusion/expr/src/logical_plan/statement.rs

```diff
@@ -148,3 +164,32 @@ pub struct SetVariable {
     /// Dummy schema
     pub schema: DFSchemaRef,
 }
+#[derive(Clone, PartialEq, Eq, Hash, Debug)]
+pub struct CreateFunction {
+    // TODO: There is open question should we expose sqlparser types or redefine them here?
```

Comment:
I think that this approach (redefine the types) is more consistent with the rest of the DataFusion codebase, though it is an unfortunate amount of replication